### ISSUE TITLE
Ethereum Basics YT Series

### ISSUE THREAD

**Issue by Saumay-Agrawal** [ethereum/ethereum-org-website#13427]
### Is your feature request related to a problem? Please describe.

While the developer docs on ethereum.org are great, I couldn't find any easy to follow video series on Youtube explaining the basics of how ethereum works under the hood. 

### Describe the solution you'd like

I have a created YT series that explains the basic concepts about the architecture of ethereum. 

This series is based on the developer docs (foundational topics section), and is supposed to complement them. Reading the documentation directly could be overwhelming for some aspiring dapp developers. I've designed and structured this video series so that it covers the foundational topics at a high level, and makes it easier for a developer to deep dive in the docs.

The series is free to access for anyone on Youtube. It's live at - https://youtube.com/playlist?list=PLqgutSGloqiJyyoL0zvLVFPS-GMD2wKa5&si=kZTf5I7PKGTXDsOZ

I'm still doing the final edits on a few videos, and they will be live soon (within next few weeks). 

I hope the ethereum community finds it valuable enough to be added to the website, so that more developers can benefit from it. 

### Describe alternatives you've considered

There are a few YT video series explaining the basics of solidity & how to build certain dapps using Hardhat or Foundry, but I couldn't find any that explains the basics of ethereum network's architecture

### Would you like to work on this issue?

- [ ] Yes
- [ ] No

**Comment by wackerow** [comment]
Very cool, this is great @Saumay-Agrawal! What did you have in mind here? Linking out to the playlist, individual videos, or embedding in the docs? cc: @konopkja Any thoughts/preferences here?

**Comment by konopkja** [comment]
Yeah this looks great, id suggest maybe start by including one link as a podcast version of the section in the  Overview or intro to Ethereum pages

**Comment by konopkja** [comment]
probably embedding would be best isntead of providing just hyperlink

**Comment by Saumay-Agrawal** [comment]
Hi Paul & Jakub!

As the video series is designed to make it easy for devs to follow the docs, I'm thinking we can include a small paragraph like: "If you are someone who learns better through videos, here is a video series covering these foundational topics" followed by an embedding to the intro video of the series. 

As Jakub already said, we can do it on either of the Overview or Intro to Ethereum pages. 

Or maybe we can add a hyperlink to the playlist on the Overview page and add the embedding of first video on the Intro to Ethereum page? That should look neat!

lmk what you guys think about this.

**Comment by Saumay-Agrawal** [comment]
hi @wackerow & @konopkja, just checking on this in case previous message got missed.

**Comment by jackabald** [comment]
Hey @Saumay-Agrawal I can work on adding your video series to the website. Please assign me the issue and I'll send a PR as soon as I am done :)

**Comment by Saumay-Agrawal** [comment]
Hey @jackabald, thanks that would be great! However, I don't think I've the necessary privileges to do that. :(

Maybe @konopkja or @wackerow can help here.🤞

**Comment by Saumay-Agrawal** [comment]
Hi @wackerow, @konopkja & @jackabald! 

Just checking in on this thread. Let me know if I can help in any way.

**Comment by wackerow** [comment]
closed with #13563, thanks folks!
